### PR TITLE
Attempt at seperating the overmap from station at backend code

### PR DESCRIPTION
**Pull request by Jalleo**
## Why It's Good For The Game
KMC wanted it done
## Changelog
:cl:
code: touched mapping.dm to try and seperate the ship and overmap code (NULL RODS WONT GO FLYING ON A OVERMAP FOR A CHANGE!)
/:cl:

### CODE REVIEW

**Comment by Jalleo** [comment]
Current status. It works and compiles. The problem is that the FTL is now broken. The specific line is probably
https://github.com/DDMers/NSV13/blob/02bf2eff8266b4d010247757d339e720fe45886b/nsv13/code/modules/overmap/ftl.dm#L204

File: code/datums/map_config.dm [hunk 1]
```diff
@@ -143,6 +143,7 @@
 			if (!fexists("_maps/[map_path]/[file]"))
 				log_world("Map file ([map_path]/[file]) does not exist!")
 				return
+
```
Comment (Jalleo) [inline code/datums/map_config.dm]: Or not I was just tired

**Comment by Kmc2000** [comment]
Supremely based jalleo

Can you mod the other JSONS for me?

**Comment by Jalleo** [comment]
I will only change the ones that are in active use and are playable otherwise they are not going to be active until the rework or fixing of issues for those maps. Better to force a focus on that. I am starting on the aegis already.

**Pull request merged**